sourcery_analytics/main.py

```diff
@@ -161,8 +163,7 @@ def cli_assess(
         console.print(f"[bold red]Found {count} errors.")
         raise typer.Exit(1)
 
-    console.print("[bold green]Assessment Complete")
-    console.print("[green]No issues found.")
+    console.print("[bold green]Assessment Complete", "[green]No issues found.")
```

Comment:
**nitpick (code_refinement):** Combine console print statements for cleaner output.

Combining the two messages into a single `console.print` call in `cli_assess` is a neat way to streamline the output. However, ensure that the intended formatting is preserved, as this change might concatenate the messages without the intended newline or separation.

```suggestion
    console.print("[bold green]Assessment Complete\n[green]No issues found.")
```